### ISSUE TITLE
错误:找不到cookie类,请确保已使用官方版网易云

### ISSUE THREAD

**Issue by cabalistljt** [nining377/dolby_beta#340]
网易云版本:安卓 8.10.30,官网下载的版本
大喇叭版本:3.5.4,在LSP仓库下载的版本

**Comment by mario6714** [comment]
Hi, one question, you use lspatch or lsposed?

**Comment by cabalistljt** [comment]
lsposed

**Comment by mario6714** [comment]
> lsposed

Ok, thanks

**Comment by ITManCHINA** [comment]
> 网易云版本:安卓 8.10.30,官网下载的版本 大喇叭版本:3.5.4,在LSP仓库下载的版本

是这样，大喇叭3.5.4最高仅支持8.8.45版本的网易云，版本高了就会报Cookie类的错误（除非作者更新大喇叭的版本咯）

**Comment by cabalistljt** [comment]
> > 网易云版本:安卓 8.10.30,官网下载的版本 大喇叭版本:3.5.4,在LSP仓库下载的版本
> 
> 是这样，大喇叭3.5.4最高仅支持8.8.45版本的网易云，版本高了就会报Cookie类的错误（除非作者更新大喇叭的版本咯）

感谢,使用8.8.45版本网易云之后可以使用了,但是某些版权歌曲会出现错误提示aggregaterror all promises were rejected,有些歌可以听有些不行,不知道你是否知道是什么问题呢?

**Comment by ITManCHINA** [comment]
> > > 网易云版本:安卓 8.10.30,官网下载的版本 大喇叭版本:3.5.4,在LSP仓库下载的版本
> > 
> > 
> > 是这样，大喇叭3.5.4最高仅支持8.8.45版本的网易云，版本高了就会报Cookie类的错误（除非作者更新大喇叭的版本咯）
> 
> 感谢,使用8.8.45版本网易云之后可以使用了,但是某些版权歌曲会出现错误提示aggregaterror all promises were rejected,有些歌可以听有些不行,不知道你是否知道是什么问题呢?

如果你最近有在关注[luci-app-unblockneteasemusic](https://github.com/UnblockNeteaseMusic/luci-app-unblockneteasemusic)项目的话就知道了，kuwo和pyncmd源最近改变了获取歌曲链接的方式，导致这两个源在杜比大喇叭上基本不可用

**Comment by cabalistljt** [comment]
> > > > 网易云版本:安卓 8.10.30,官网下载的版本 大喇叭版本:3.5.4,在LSP仓库下载的版本
> > > 
> > > 
> > > 是这样，大喇叭3.5.4最高仅支持8.8.45版本的网易云，版本高了就会报Cookie类的错误（除非作者更新大喇叭的版本咯）
> > 
> > 
> > 感谢,使用8.8.45版本网易云之后可以使用了,但是某些版权歌曲会出现错误提示aggregaterror all promises were rejected,有些歌可以听有些不行,不知道你是否知道是什么问题呢?
> 
> 如果你最近有在关注[luci-app-unblockneteasemusic](https://github.com/UnblockNeteaseMusic/luci-app-unblockneteasemusic)项目的话就知道了，kuwo和pyncmd源最近改变了获取歌曲链接的方式，导致这两个源在杜比大喇叭上基本不可用

安装默认的为这两个音源,请问大喇叭除了qq还能使用什么音源呢?对音质没有要求,更希望为准确音源,感谢你的耐心回复

**Comment by ITManCHINA** [comment]
自定义音源输入`pyncmd kuwo qq migu`应该就没问题了，但是准确性来说的话qq和migu这俩都没法保证高准确性啊😂

**Comment by cabalistljt** [comment]
音源设置仅为qq也会弹出aggregaterror all promises were rejected,是我哪里没有设置正确吗?

**Comment by ITManCHINA** [comment]
> 音源设置仅为qq也会弹出aggregaterror all promises were rejected,是我哪里没有设置正确吗?

如果音源正常工作请关闭`开启Hook警告`的开关并无视此消息

**Comment by mario6714** [comment]
https://github.com/nining377/dolby_beta/pull/342

**Comment by ITManCHINA** [comment]
> #342

Freaking awesome! Thanks to mention it!😄

**Comment by mario6714** [comment]
> > #342
> 
> Freaking awesome! Thanks to mention it!😄

You need get a kuwo cookie

**Comment by ITManCHINA** [comment]
> > > #342
> > 
> > 
> > Freaking awesome! Thanks to mention it!😄
> 
> You need get a kuwo cookie

Yep, I got it and it‘s already in used by [luci-app-unblockneteasemusic](https://github.com/UnblockNeteaseMusic/luci-app-unblockneteasemusic)

**Comment by mario6714** [comment]
> > > > #342
> > > 
> > > 
> > > Freaking awesome! Thanks to mention it!😄
> > 
> > 
> > You need get a kuwo cookie
> 
> Yep, I got it and it‘s already in used by [luci-app-unblockneteasemusic](https://github.com/UnblockNeteaseMusic/luci-app-unblockneteasemusic)

I need a hardware for use this "luci-app-unblockneteasemusic" right?

**Comment by ITManCHINA** [comment]
> > > > > #342
> > > > 
> > > > 
> > > > Freaking awesome! Thanks to mention it!😄
> > > 
> > > 
> > > You need get a kuwo cookie
> > 
> > 
> > Yep, I got it and it‘s already in used by [luci-app-unblockneteasemusic](https://github.com/UnblockNeteaseMusic/luci-app-unblockneteasemusic)
> 
> I need a hardware for use this "luci-app-unblockneteasemusic" right?

Yep, you need an openwrt device to install it.

**Comment by cabalistljt** [comment]
> 
QAQ程序小白没看懂这个是怎么用的...有相关教程学习一下吗```

**Comment by cabalistljt** [comment]
> > > > > > #342
> > > > > 
> > > > > 
> > > > > Freaking awesome! Thanks to mention it!😄
> > > > 
> > > > 
> > > > You need get a kuwo cookie
> > > 
> > > 
> > > Yep, I got it and it‘s already in used by [luci-app-unblockneteasemusic](https://github.com/UnblockNeteaseMusic/luci-app-unblockneteasemusic)
> > 
> > 
> > I need a hardware for use this "luci-app-unblockneteasemusic" right?
> 
> Yep, you need an openwrt device to install it.

QAQ程序小白没看懂这个是怎么用的...有相关教程学习一下吗```

**Comment by mario6714** [comment]
> > > > > > #342
> > > > > 
> > > > > 
> > > > > Freaking awesome! Thanks to mention it!😄
> > > > 
> > > > 
> > > > You need get a kuwo cookie
> > > 
> > > 
> > > Yep, I got it and it‘s already in used by [luci-app-unblockneteasemusic](https://github.com/UnblockNeteaseMusic/luci-app-unblockneteasemusic)
> > 
> > 
> > I need a hardware for use this "luci-app-unblockneteasemusic" right?
> 
> Yep, you need an openwrt device to install it.

Something like this work?

https://www.amazon.com.mx/GL-MT300N-V2-Converter-Pre-installed-Performance-Compatible/dp/B073TSK26W/ref=sr_1_1?__mk_es_MX=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=1ITNL6SG1RCE4&keywords=GL-MT300N&qid=1691217362&sprefix=gl-mt300n%2Caps%2C331&sr=8-1
